src/js/modules/infragistics.ui.editors.js

```diff
@@ -38,36 +38,168 @@
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					width : 200
+				});
+
+				//Get
+				var width = $(".selector").%%WidgetName%%("option", "width");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "width", 200);
+				```
 				string The widget width can be set in pixels (px) and percentage (%).
 				number The widget width can be set as a number in pixels.
 				null type="object" will stretch to fit data, if no other widths are defined.
 			*/
 			width: null,
 			/* type="string|number|null" Gets/Sets how the height of the control can be set.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					height : 25
+				});
+
+				//Get
+				var height = $(".selector").%%WidgetName%%("option", "height");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "height", 25);
+				```
 				string The height can be set in pixels (px) and percentage (%).
 				number The height can be set as a number in pixels.
 				null type="object" will fit the editor inside its parent container, if no other heights are defined.
 			*/
 			height: null,
-			/* type="object" Gets/Sets value in editor. The effect of setting/getting that option depends on type of editor and on dataMode options for every type of editor.*/
+			/* type="object" Gets/Sets value in editor. The effect of setting/getting that option depends on type of editor and on dataMode options for every type of editor.
+			```
+			//Initialize
+			$(".selector").%%WidgetName%%({
+				value : "Some text"
+			});
+
+			//Get
+			var value = $(".selector").%%WidgetName%%("option", "value");
+
+			//Set
+			$(".selector").%%WidgetName%%("option", "value", "Some text");
+
+			```
+			*/
 			value: null,
-			/* type="number" Gets/Sets value in tabIndex for editor. */
+			/* type="number" Gets/Sets value in tabIndex for editor.
+			  ```
+			  //Initialize
+			  $('.selector').%%WidgetName%%({
+				  tabIndex: 1
+			  });
+
+			  //Get
+			  var tabIndex = $(".selector").%%WidgetName%%("option", "tabIndex");
+
+			  //Set
+			  $(".selector").%%WidgetName%%("option", "tabIndex", 1);
+			  ```
+			*/
 			tabIndex: null,
 			/* type="bool" Gets/Sets ability to prevent null value.
 				If that option is false, and editor has no value, then value is set to an empty string.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					allowNullValue : false
+				});
+
+				//Get
+				var allowNullValue = $(".selector").%%WidgetName%%("option", "allowNullValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "allowNullValue", false);
+				```
 			*/
 			allowNullValue: false,
-			/* type="string|number|null" Gets/Sets the representation of null value. In case of default the value for the input is set to null, which makes the input to hold an empty string */
+			/* type="string|number|null" Gets/Sets the representation of null value. In case of default the value for the input is set to null, which makes the input to hold an empty string
```

Comment:
When the editor is cleared (by clear button on cleared and blurred) depending on this option the value of the editor is set to nullValue option, or empty string. In the derived widgets such as Numeric type there is another comment explaining the different behavior.